server/src/main/java/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -2075,6 +2075,7 @@ public CapacityVO getStoragePoolUsedStats(Long poolId, Long clusterId, Long podI
         if (poolId != null) {
             sc.addAnd("hostOrPoolId", SearchCriteria.Op.EQ, poolId);
         }
+        sc.addAnd("parent", SearchCriteria.Op.EQ, 0);
```

Comment:
@DK101010 `parent` is of Long type in `StoragePoolVO`
```suggestion
        sc.addAnd("parent", SearchCriteria.Op.EQ, 0L);
```